frontend/lib/Views/AdminPage/utils.dart

```diff
@@ -40,3 +40,67 @@ Widget cancelButton(BuildContext context) => OutlinedButton(
         ),
       ),
     );
+
+
+PreferredSize buildAdminAppBar(String bottomText) => PreferredSize(
+  preferredSize: const Size.fromHeight(120),
+  child: AppBar(
+    iconTheme: const IconThemeData(
+      color: adminGreenColor,
+    ),
+    centerTitle: true,
+    toolbarHeight: 120,
+    title: Container(
+      padding: const EdgeInsets.only(top: 20, bottom: 10),
+      child: const Text("Administrator panel"),
+    ),
+    backgroundColor: Colors.white,
+    titleTextStyle: const TextStyle(
+      color: Colors.black,
+      fontFamily: 'Ubuntu Condensed',
+      fontSize: 40,
+      fontWeight: FontWeight.w500,
+    ),
+    titleSpacing: 20,
+    bottom: bottomText != "" ?  PreferredSize(
+      preferredSize: const Size.fromHeight(20),
+      child: Container(
+        padding: const EdgeInsets.only(bottom: 10, top: 5),
+        child: Text(
+          bottomText,
+          style: const TextStyle(fontSize: 25),
+        ),
+      ),
+    ) : null,
+  ),
+);
+
+Container buildDeleteContainer(Function onDeletePressed, arguments) => Container(
```

Comment:
Przydałoby się dodać do `onDeletePressed`, jakie argumenty ta funkcja przyjmuje i co zwraca

Reply:
not sure, dla kazdego widoku ta funkcja moze przyjmowac inne argumenty

Reply:
Why? To powinien być zwykły callback, który nic nie przyjmuje i zwraca void